1주차/스택.py

```diff
@@ -0,0 +1,40 @@
+def push(X):
+    my_stack.append(X)
+
+def top():
+    if len(my_stack) >= 1:
+        return my_stack[-1]
+    else:
+        return -1
+
+def size():
+    return len(my_stack)
+
+def empty():
+    if len(my_stack) == 0:
```

Comment:
```suggestion
    if not my_stack:
```
마찬가지로 여긴 이렇게 쓸 수도 있겠죠